tts_config/sample/tts/scripts/opennebula.py

```diff
@@ -178,19 +179,20 @@ def main():
             jobject = json.loads(str(base64.urlsafe_b64decode(json_data)))
 
             action = jobject['action']
+            user_info = jobject['user_info']
+            oidc = user_info['oidc']
+            iss = urlparse.urlparse(oidc['iss'])
+            iss_host = iss[1]
+            username = "%s_%s_%s" % (USER_PREFIX, iss_host, oidc['sub'])
 
             if action == "request":
-                user_info = jobject['user_info']
-                oidc = user_info['oidc']
-                iss = urlparse.urlparse(oidc['iss'])
-                iss_host = iss[1]
-                username = "%s_%s" % (iss_host, oidc['sub'])
-
                 print create_user(username, USERS_GROUP, oidc)
             elif action == "revoke":
                 state = jobject['cred_state']
-
-                print revoke_user(state)
+                if state != username:
```

Comment:
I have added this for security purposes. To avoid that someone tries to delete any user of the opennebula server. In this way only a user with a correct token can remove his user and not others.